cognee/shared/CodeGraphEntities.py

```diff
@@ -1,13 +1,11 @@
 from typing import List, Optional
-
 from cognee.infrastructure.engine import DataPoint
 
 
 class Repository(DataPoint):
     __tablename__ = "Repository"
     path: str
     _metadata: dict = {
-        "index_fields": ["source_code"],
```

Comment:
The index_data_points function in add_data_points throws a KeyError: 'index_fields' on my side. Maybe you handled it there and didn't add the changes to the PR?

Reply:
nevertheless, "index_fields": [], solves it